### PR TITLE
Final

### PR DESCRIPTION
**Pull request by loqit**

### CODE REVIEW

**Review by gmltA — CHANGES_REQUESTED** [review]
Everything's fine.

Can you add a [skeleton](https://material-ui.com/components/skeleton/) for when your boards and items are being loaded from the remote?

File: public/js/boards.js [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
 document.addEventListener("DOMContentLoaded", function() {
     userId = localStorage['userId'] || '';
+
     getBoards(userId, async board => {
         if (boards[board.id] === undefined) { // Display board only once
             boards[board.id] = board;
             await displayBoard(board);
         }
     });
+    setTimeout(function(){ removeElementsByClass('board-skeleton') }, 300);
```
Comment (gmltA) [inline public/js/boards.js]: Uhm you should display it while your data is being loaded. Not for some arbitrary period.

**Review by gmltA — CHANGES_REQUESTED** [review]

**Review by gmltA — APPROVED** [review]
Good. That's a <kbd>9</kbd>. And the resulting mark will be <kbd>9</kbd> too.

**Comment by loqit** [comment]
> Good. That's a <kbd>9</kbd>. And the resulting mark will be <kbd>9</kbd> too.

Thanks

**Pull request merged**